Tests/UnitTests/EdgeHitProcessorTests.swift

```diff
@@ -641,24 +738,38 @@ class EdgeHitProcessorTests: XCTestCase {
                                         getImplementationDetails: { return nil },
                                         getLocationHint: { return nil })
 
-        mockNetworkService?.connectAsyncMockReturnConnection = HttpConnection(data: "{}".data(using: .utf8), response: HTTPURLResponse(url: url, statusCode: 200, httpVersion: nil, headerFields: nil), error: nil)
+        mockNetworkService.setMockResponse(
+            url: INTERACT_ENDPOINT_PROD,
+            httpMethod: .post,
+            responseConnection: HttpConnection(
+                data: "{}".data(using: .utf8),
+                response: HTTPURLResponse(
+                    url: url,
+                    statusCode: 200,
+                    httpVersion: nil,
+                    headerFields: nil),
+                error: nil))
 
         let edgeEntity = EdgeDataEntity(event: experienceEvent, identityMap: [:])
         let entity = DataEntity(uniqueIdentifier: "test-uuid", timestamp: Date(), data: try? JSONEncoder().encode(edgeEntity))
 
         // test
         assertProcessHit(entity: entity, sendsNetworkRequest: true, returns: true)
 
-        guard let requestPayload = mockNetworkService?.connectAsyncCalledWithNetworkRequest?.connectPayload else {
-            XCTFail("unexpected nil request connect payload")
+        guard let networkRequest = mockNetworkService.getNetworkRequestsWith(url: INTERACT_ENDPOINT_PROD, httpMethod: .post).first else {
+            XCTFail("Unable to find valid network request.")
             return
         }
 
-        let payload = asFlattenDictionary(data: requestPayload)
+        let expectedJSON = #"""
+        {
+          "xdm": {
+            "implementationDetails": null
+          }
+        }
```

Comment:
Does this mean that "implementationDetails" is not added to the "xdm" object, or that the field "implementationDetails" is added but has a value of "null"?  I would expect "implementationDetails" to not be included in the request.